src/api/test/models/event_test.rb

```diff
@@ -84,64 +22,4 @@ def groups_for_event(e)
     CleanupEvents.new.perform
     assert Event::Base.count != firstcount, 'now its gone'
   end
-
-  test 'maintainer mails for build failure' do
-    # for this test we don't want fixtures to interfere
-    EventSubscription.delete_all
-
-    # just one subsciption
-    EventSubscription.create(eventtype: 'Event::BuildFail', receiver_role: :maintainer, user: users(:Iggy), channel: :instant_email, enabled: true)
-
-    assert_equal ['Iggy'], users_for_event(events(:build_failure_for_iggy))
-  end
-
-  test 'reader mails for build failure' do
-    # for this test we don't want fixtures to interfere
-    EventSubscription.delete_all
-
-    # just one subsciption
-    EventSubscription.create(eventtype: 'Event::BuildFail', receiver_role: :reader, user: users(:fred), channel: :instant_email, enabled: true)
-
-    assert_equal ['fred'], users_for_event(events(:build_failure_for_reader))
-  end
-
-  test 'maintainer mails for source service fail' do
-    # for this test we don't want fixtures to interfere
-    EventSubscription.delete_all
-
-    # just one subsciption
-    EventSubscription.create(eventtype: 'Event::ServiceFail', receiver_role: :maintainer, user: users(:Iggy), channel: :instant_email, enabled: true)
-
-    assert_equal ['Iggy'], users_for_event(events(:service_failure_for_iggy))
-  end
-
-  test 'package maintainer mail' do
```

Comment:
@dmarcoux not sure if thats a problem, but the specs mentioned in the commit message don't cover the mail stuff. I would say sending out the emails and events are not related and should be tested independent. Looking at https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/jobs/send_event_emails_job_spec.rb I dont see the email channel covered

Reply:
This [line](https://github.com/openSUSE/open-build-service/blob/888a1037283bd7019055c3104617cd89b93ca3bd/src/api/spec/jobs/send_event_emails_job_spec.rb#L21) is with the `instant_email` channel as this is the default channel in the [factory](https://github.com/openSUSE/open-build-service/blob/888a1037283bd7019055c3104617cd89b93ca3bd/src/api/spec/factories/event_subscriptions.rb#L8). As for emails sent, this is covered in those [lines](https://github.com/openSUSE/open-build-service/blob/888a1037283bd7019055c3104617cd89b93ca3bd/src/api/spec/jobs/send_event_emails_job_spec.rb#L30-L35).